components/engine-hdb/src/main/java/com/codbex/kronos/engine/hdb/processors/HDBSynonymCreateProcessor.java

```diff
@@ -57,6 +57,10 @@ public void execute(Connection connection, HDBSynonymGroup synonymModel) throws
 	      try {
 	        String synonymSchema = null != entry.getValue().getSchema() ? entry.getValue().getSchema() : connection.getMetaData().getUserName();
 	        if (!SqlFactory.getNative(connection).exists(connection, synonymSchema, entry.getKey(), DatabaseArtifactTypes.SYNONYM)) {
+				// TODO: Add Support for public synonym creation
+				// "CREATE SYNONYM "PUBLIC"."com.codbex.test.data::Test.Table1234" FOR "SCHEMA_NAME"."com.codbex.test.data::Test.Table1234""
+				// ->
+				// "CREATE PUBLIC SYNONYM "com.codbex.test.data::Test.Table1234" FOR "SCHEMA_NAME"."com.codbex.test.data::Test.Table1234""
```

Comment:
Support for creation of `publicSynonym()` to be added or the an exception should not be thrown.